src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
It's just a minor detail, as it doesn't matter in this particular case, but I'm going to write it anyway just as a tip for the future - in case of big enum classes calling `values()` method multiple times can be inefficient, so a good way to avoid this issue would be to call `Colors.values()` once and store the result in a variable inside of this method, so we can use it according to the needs.